### ISSUE TITLE
ESP stuck in restart loop

### ISSUE THREAD

**Issue by vyruz1986** [SensorsIot/IOTAppstory.com#12]
I published my first app on IOTAppstory.com (woohoo! 👍 ).
One issue I had during the development of the app, and now have again on an ESP which worked yesterday with the app on it, but today I brought it to another place (which did not have my home WiFi network).

The symptom is that the ESP keeps continously rebooting with the following logs on the serial port:
```
Reading Config
Configuration found
..................................................
No Connection. Try to connect with saved PW
..................................................Going into Configuration Mode

 ets Jan  8 2013,rst cause:2, boot mode:(3,6)

load 0x4010f000, len 1384, room 16 
tail 8
chksum 0x2d
csum 0x2d
v3de0c112
~ld


Reading Config
Configuration found
..................................................
No Connection. Try to connect with saved PW
..................................................Going into Configuration Mode

 ets Jan  8 2013,rst cause:2, boot mode:(3,6)

load 0x4010f000, len 1384, room 16 
tail 8
chksum 0x2d
csum 0x2d
v3de0c112
~ld
```

The only which solves this situation, is flashing a completely different sketch, which logs on to a hard-coded wifi network using the `WiFi.begin(ssid, password);` function.
After that i can flash the IOTAppstory sketch again and it seems to work fine.

I am thinking this weekend the problem occurred when I changed/added paramers to the config struct, which is loaded into EEPROM as far as I know, could it be that changing the struct with additional parameters messes up the struct in the EEPROM and causes the app to be unable to read its config?

**Comment by SensorsIot** [comment]
Maybe you add a print all the variables fro the eeprom. Then, you see what happens. It is well possible that it has a a valid, but wrong PW in the EEPROM. I think, I changed the connectNetwork() function in the new ESP_helpers.h. Maybe you download it. But be aware, that I also changed some stuff which is now in the ESP_helpers file and can be deleted in your main file. The compiler will show it to you (with error messages)...
I think, the ESP_helpers file in the Loader directory is still the old one, so take the one in the ESP_helpers directory.
